src/webview/webViewEventHandlers.js

```diff
@@ -251,6 +259,18 @@ export const handleMessageListEvent = (props: Props, _: GetText, event: MessageL
       logging.error(`WebView exception: ${JSON.stringify(event.details)}`);
       break;
 
+    case 'mention': {
```

Comment:
This should go with the rest of the normal event types, and above the special events like `'debug'` and `'error'`.